src/ce/tice.h

```diff
@@ -919,6 +925,33 @@ int os_PushErrorHandler(void) __attribute__((returns_twice));
  */
 void os_PopErrorHandler(void);
 
+/**
+ * Runs a program that exists on the calculator.
+ * Note that this will destroy the currently running program, requiring you to
+ * save any data as needed. This program has an optional callback that will be
+ * executed when the called program finishes, which can be used to rebuild the
+ * program state. Additionally, program context information can be safely
+ * stored by using the extra user data arguments, which will then be delivered
+ * to the callback.
+ *
+ * @param prgm Name of program to execute. The type should be the first byte of
+ * the string.
```

Comment:
I don't see a use case for including the type byte. Users wanting to run BASIC from an appvar seems like an incredibly rare occurrence, and in my opinion it would be better to add a function to fileioc for changing the type of a variable so that users can change the appvar to a program if they want to run code from it.

Reply:
I would say evaluating a function, but I imagine that is better served by another routine since functions can't call asm programs so you don't need to tear everything down.

Reply:
I disagree.

Reply:
Now I agree.